package.json

```diff
@@ -60,6 +61,7 @@
     "@typescript-eslint/parser": "^5.51.0",
     "babel-plugin-inline-import": "^3.0.0",
     "coveralls": "^3.1.1",
+    "cpx": "^1.5.0",
```

Comment:
Why cpx rather than pure cp?
AFACIT, cpx would be useful for watching file changes, but that's not needed here.